USER_GUIDE.md

```diff
@@ -124,4 +124,72 @@ func main() {
     fmt.Println("deleting index", deleteIndexResponse)
 }
 
-```
\ No newline at end of file
+```
+
+## How to use IAMs as authentication method
+
+Before starting, we strongly recommend to read the full AWS documentation regarding IAMs for Opensearch.
+See https://docs.aws.amazon.com/opensearch-service/latest/developerguide/ac.html
```

Comment:
I would recommend linking off to the developer guide with some text, not just a hyperlink.

See [Identity and Access Management in Amazon OpenSearch Service](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/ac.html).